src/specBuilder/bar/stackedBarUtils.ts

```diff
@@ -79,6 +79,7 @@ export const getStackedBackgroundBar = (props: BarSpecProps): RectMark => {
 			},
 			update: {
 				...getStackedDimensionEncodings(props),
+				cursor: getCursor(props.children, props),
```

Comment:
did we test if it works without the cursor defined here? Previously we only had `cursor` defined in `getBarUpdateEncodings` in `barUtils.ts`. This is a utility that both `stacked and dodged bars use. So I think by updating that spot (which you already did), then you shouldn't need to define it here or on line 117.

Reply:
But I could be wrong

Reply:
I defined them here too because there are some test cases for that expect the cursor to be defined as undefined by default, which they aren't without this definition for some reason
![Screenshot 2024-09-04 at 9 10 07 AM](https://github.com/user-attachments/assets/a8bdcbd5-39b6-49d8-9769-b0a2bdb3c0c4)

Reply:
```suggestion
```